RecoJets/Configuration/python/RecoPFJets_cff.py

```diff
@@ -77,30 +67,18 @@
                            fixedGridRhoFastjetCentral,
                            fixedGridRhoFastjetCentralChargedPileUp,
                            fixedGridRhoFastjetCentralNeutral,
-                           ak4PFJets,ak8PFJets,
+                           ak4PFJets,
+			   ak8PFJets,
                            pfNoPileUpJMETask,
-                           ak8PFJetsCHS,
-                           ak8PFJetsCHSPruned,
-                           ak8PFJetsCHSFiltered,
-                           ak8PFJetsCHSTrimmed,
-                           ak8PFJetsCHSSoftDrop,
-                           ak4PFJetsCHS,                                                      
-                           ak8PFJetsCHS,
-                           ak8PFJetsCHSPruned,
-                           ak8PFJetsCHSFiltered,
-                           ak8PFJetsCHSTrimmed,
-                           ak8PFJetsCHSSoftDrop,
-                           ak8PFJetsCHSConstituents,
-                           ak8PFJetsCHSPrunedMass,
-                           ak8PFJetsCHSTrimmedMass,
-                           ak8PFJetsCHSSoftDropMass,
-                           ak8PFJetsCHSFilteredMass,
-                           ca15PFJetsCHSMassDropFiltered,
-                           ca15PFJetsCHSFiltered,
-                           ak8PFJetsCS,
-                           ak8PFJetsCSConstituents,
+                           ak4PFJetsCHS,
                            puppi,
                            ak4PFJetsPuppi,
+                           ak8PFJetsPuppi,
+                           ak8PFJetsPuppiSoftDrop,
+                           ak8PFJetsPuppiConstituents,
+                           ak8PFJetsPuppiSoftDropMass,
+                           ak8PFJetsCS,
+                           ak8PFJetsCSConstituents,                           
                            softKiller,
                            ak4PFJetsSK
     )
```

Comment:
I suppose ak8PFJetsCHS should be removed from recoPFJetsHITask (below); no reason to run it if the event content is not going to keep it